snooty/n.py

```diff
@@ -334,6 +334,11 @@ class Target(Parent[Node]):
     refuri: Optional[str]
     html_id: Optional[str]
 
+    @property
+    def identifier(self) -> Tuple[str, str]:
```

Comment:
Is it plausible that `Target.identifier()` could be accidentally conflated with `TargetIdentifier`?

Reply:
This is actually dead code! Removing